packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -729,6 +729,12 @@ $Calendar-dayPicker-margin: 10px;
   .yearComponents {
     margin-top: 2px;
   }
+  .divider {
+    display: none;
+  }
+  .goTodayInlineMonth {
+    top: initial;
```

Comment:
I have never seen `initial` used. Why are you using it?

Reply:
The `Go to Today` text overlaps with the calendar, looking like this (not the original bug reported).
![image](https://user-images.githubusercontent.com/8922176/60219568-c2271b00-9827-11e9-860f-05ae405aa1fb.png)


It needs styling on `top` to fix this. I changed it to `top: auto;`
![image](https://user-images.githubusercontent.com/8922176/60219560-bb98a380-9827-11e9-94dd-ebed077a38f9.png)

Reply:
Above behavior is seen only on mobile view, as reported in the issue.